lib/wptfyi/client.go

```diff
@@ -84,3 +100,13 @@ func (w HTTPClient) GetRuns(
 
 	return allRuns, nil
 }
+
+func is404Error(err error) bool {
+	// nolint:lll // WONTFIX: commit URL is useful
```

Comment:
What's the `lll`?

Reply:
It's the long length linter ([docs](https://golangci-lint.run/usage/linters/#lll)). It was reporting based on my comment which has a long URL.